Strapi API.postman_collection.json

```diff
@@ -0,0 +1,192 @@
+{
+	"info": {
+		"_postman_id": "65e9ffe8-7844-4b04-9604-fbf77063e7b2",
+		"name": "Strapi API",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Time Capsule Requests",
+			"item": [
+				{
+					"name": "GET time capsules",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{strapiApiUrl}}/users/:id/time-capsules",
+							"host": [
+								"{{strapiApiUrl}}"
+							],
+							"path": [
+								"users",
+								":id",
+								"time-capsules"
+							],
+							"variable": [
+								{
+									"key": "id",
```

Comment:
We didn't seed the database. Where is this value coming from?

Reply:
Oh, this is from postman. I set up a path variable called id and gave it the value 96. This is so when I make the request through postman it'll pass 96 to the get request `/users/96/time-capsules`